src/components/ListJobs.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="list-jobs">
+    <v-list width="100%">
+      <v-list-item v-for="job in jobs" :key="job.infos.jobID"  :to="`/editor/${job.infos.jobID}`">
+        <v-list-item-icon>
+         <v-icon small v-text="`mdi-checkbox-blank-circle`" :color="iconColor(job.infos.status)"></v-icon>
+       </v-list-item-icon>
+       <v-list-item-content>
+        <v-list-item-title v-text="job.infos.jobID"></v-list-item-title>
+      </v-list-item-content>
+      </v-list-item>
+    </v-list>
+  </div>
+</template>
+
+<script>
+const colors = {
+  running: '#009688',
+  failed: '#965300',
+  accepted: '#009688',
+  successful: '#4CAF50'
+}
+
+export default {
+  data () {
+    return {
+      jobs: []
+    }
+  },
+  mounted () {
+    this.getJobs()
+  },
+  methods: {
+    getJobs () {
```

Comment:
Can you move the getJobx function in wps folder similar like the other functions. Then call this function in the component.  

Suggestion: While the function is loading the response there we can show a circle that is loading. I noticed that it is taking time to show the results.